stdlib/InteractiveUtils/src/editless.jl

```diff
@@ -148,6 +149,9 @@ function define_default_editors()
     define_editor(Any["micro", "kak"]; wait=true) do cmd, path, line, column
         `$cmd +$line $path`
     end
+    define_editor(Any["hx", "helix"]; wait=true) do cmd, path, line, column
```

Comment:
I know you're copying from another line but I'm not sure why we need the `Any` here.

Reply:
Me neither - this array ends up being passed to `editor_matches` right above this function, which doesn't seem to care about the type either way.

Reply:
If you look a bit below there is an array of Strings without `Any`. So I would say it's not needed, there is one which has Strings and Regexes which might have led to other people just assuming it was the way.